content/en/tracing/guide/ddsketch_trace_metrics.md

```diff
@@ -41,9 +41,37 @@ The APM Service and Resource pages use this metric type automatically. This mean
 - Distribution Metrics now power APM Service and Resource pages as Datadog rolls this out for more customers.
 - Datadog has migrated all manual queries from dashboards and monitors to the new metrics automatically on your behalf, while retaining the existing metrics to support historical views.
 
+**I'm using Terraform, what does this change mean for me?**
+- The existing metrics are not going away, this means that your Terraform definitions are still in place and still work well.
```

Comment:
```suggestion
The existing metrics are not going away; your Terraform definitions are still in place and still work.
```

Reply:
@andrewardito how did I miss this? it's not "the existing metrics are not going away". I updated this in my draft to be "The existing metrics are still around"